taskt/Core/Automation/Commands/Programs_Process Commands/RunScriptCommand.cs

```diff
@@ -1,65 +1,69 @@
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements 'Process.Start' and waits for the script/program to exit before proceeding.")]
+    [Group("Programs/Process Commands")]
+    [Description("This command runs a script or program and waits for it to exit before proceeding.")]
+
     public class RunScriptCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Enter the path to the script, or select a file.")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter a fully qualified path to the script, including the script extension.")]
-        [Attributes.PropertyAttributes.SampleUsage("**C:\\temp\\myscript.vbs**")]
-        [Attributes.PropertyAttributes.Remarks("This command differs from **Start Process** because this command blocks execution until the script has completed.  If you do not want to stop while the script executes, consider using **Start Process** instead.")]
+        [PropertyDescription("Script Path")]
+        [InputSpecification("Enter a fully qualified path to the script, including the script extension.")]
+        [SampleUsage(@"C:\temp\myscript.ps1 || {vScript} || {ProjectPath}\myscript.ps1")]
+        [Remarks("This command differs from *Start Process* because this command blocks execution until the script has completed. " +
+                 "If you do not want to stop while the script executes, consider using *Start Process* instead.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
         public string v_ScriptPath { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Script Type")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Powershell")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Python")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Default")]
-        [Attributes.PropertyAttributes.InputSpecification("Select the type of script you want to execute.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select Powershell to run ps1 files, Default to run a file through the system default.")]
-        [Attributes.PropertyAttributes.Remarks("Default executes with the system default for that file type.")]
+        [PropertyDescription("Script Type")]
+        [PropertyUISelectionOption("Default")]
+        [PropertyUISelectionOption("Powershell")]
+        [PropertyUISelectionOption("Python")]
+        [InputSpecification("Select the type of script you want to execute.")]
```

Comment:
Please add the SampleUsage and keep it empty

Reply:
done